lib/Common/Core/ConfigParser.cpp

```diff
@@ -337,19 +337,70 @@ void ConfigParser::ParseConfig(HANDLE hmod, CmdLineArgsParser &parser)
         {
             return;
         }
+
         if (_wfopen_s(&configFile, filename, _u("r, ccs=UNICODE")) != 0 || configFile == nullptr)
         {
             return;
         }
     }
+#else
+    // Two-pathed for a couple reasons
+    // 1. PAL doesn't like the ccs option passed in.
```

Comment:
When I tried this out, wfopen failed if I passed in ccs=UNICODE- I didn't dig in too deeply since this is test code and I need something up quickly to unblock myself on xplat 😄